internal/serve/httphandler/receiver_wallets_handler.go

```diff
@@ -0,0 +1,38 @@
+package httphandler
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/go/support/render/httpjson"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/data"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/db"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/serve/httperror"
+)
+
+type ReceiverWalletsHandler struct {
+	Models           *data.Models
+	DBConnectionPool db.DBConnectionPool
+}
+
+func (h ReceiverWalletsHandler) RetryInvitation(rw http.ResponseWriter, req *http.Request) {
+	ctx := req.Context()
+
+	receiverWalletID := chi.URLParam(req, "receiver_wallet_id")
+	receiverWallet, err := h.Models.ReceiverWallet.RetryInvitationSMS(ctx, receiverWalletID, h.DBConnectionPool)
+	if err != nil {
+		if errors.Is(err, data.ErrRecordNotFound) {
+			httperror.NotFound("", err, nil).Render(rw)
+			return
+		}
+		err = fmt.Errorf("finding the receiver: %w", err)
+		log.Ctx(ctx).Error(err)
+		httperror.InternalError(ctx, "", err, nil).Render(rw)
```

Comment:
Calls to `httperror.InternalError` already trigger a log-and-report error call, so this log here is redundant.

```suggestion
		httperror.InternalError(ctx, "", err, nil).Render(rw)
```

Reply:
Here are the code blocks that setup that behavior:

https://github.com/stellar/stellar-disbursement-platform-backend/blob/0aabb19139bd9c0699a53e6a6b6319ec624f9383/internal/serve/httperror/httperror.go#L119

https://github.com/stellar/stellar-disbursement-platform-backend/blob/0aabb19139bd9c0699a53e6a6b6319ec624f9383/internal/serve/serve.go#L94-L95

https://github.com/stellar/stellar-disbursement-platform-backend/blob/0aabb19139bd9c0699a53e6a6b6319ec624f9383/internal/serve/httperror/httperror.go#L25-L38